analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/TokenTypeAnalyzerTest.Classifier.cs

```diff
@@ -520,6 +520,19 @@ public class Inner { }
               }
               """, allowSemanticModel);
 
+    [DataTestMethod]
+    [DataRow("([t:String] s1, [t:String] s2)", false)]
+    [DataRow("([u:System].String s1, int i)", true)]
+    [DataRow("(([t:String] s, int i), bool b)", false)]
```

Comment:
in this cases, it's not clear how we classify `int` and `bool`. Could you please document it?